www/hugo/Portfile

```diff
@@ -3,29 +3,30 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        gohugoio hugo 0.30.2 v
+github.setup        gohugoio hugo 0.42.1 v
 revision            0
 categories          www
 platforms           darwin
-license             BSD
-maintainers         nomaintainer
+license             Apache-2
+maintainers         @cardi openmaintainer
```

Comment:
These days, a correct `maintainers` line needs to include your obfuscated email, as in:

`maintainers         {domain.com:localpart @cardi} openmaintainer`